examples/distributed_data_classification_examples/test_models.ipynb

```diff
@@ -0,0 +1,373 @@
+{
+ "cells": [
```

Comment:
It would be good if you could add a text cell at the beginning describing what the goal of the notebook is. Just a simple paragraph describing the inputs, outputs, and requirements.

Reply:
Done